active/InputValidation/InputValidation.md

```diff
@@ -0,0 +1,572 @@
+> See comments in Markdown for how to use this spec template
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+In WPF Developers had access to Input Validation via [INotifyDataErrorInfo](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.inotifydataerrorinfo?view=netframework-4.7.2) We are adding the same capability to UWP.
```

Comment:
"We are adding the same capability to UWP" exactly the same or with some differences/modifications?
If identical, what's the need for this spec?
If not identical, what are the differences? If that's this doc, please be explicit about this.

Reply:
Thanks for the feedback, How about this... _"There are differences from the previous implementations, we are building off the of lessons learned. Which will results in differences in the new model."_